tools/lxdclient/client.go

```diff
@@ -271,7 +271,7 @@ func verifyDefaultProfileBridgeConfig(client *lxd.Client, networkAPISupported bo
 		return "", errors.Trace(err)
 	}
 
-	// If the default profile doesn't have eth0 in it, then the user has messed
+	// If the default profile has eth0 in it, then the user has messed
```

Comment:
This comment contradicts the error message on line 289, which is returning an error because eth0 wasn't found.